apps/opik-backend/src/main/resources/liquibase/db-app-state/migrations/000004_add_prompt_library_tables.sql

```diff
@@ -0,0 +1,33 @@
+--liquibase formatted sql
+--changeset thiagohora:add_prompt_library_tables
+
+CREATE TABLE prompts (
```

Comment:
For idempotent migrations, better add `IF NOT EXISTS` if all these statements. In this case, it's supported in MySQL.